src/dom-updates.js

```diff
@@ -57,10 +70,52 @@ const setRoomImage = (room) => {
 
 const checkForBidet = (room) => {
   let bidet;
-  room.bidet ? bidet = 'has bidet' : bidet = 'no bidet'
+  room.bidet ? bidet = 'with bidet' : bidet = ''
   return bidet;
 }
 
+const displaySearchResults = (allBookings, allRooms, date) => {
+  hide(dashboard)
+  show(searchResults)
+
+  const availRooms = searchByDate(allBookings, allRooms, date);
+  availableRooms.innerHTML = '';
+
+  if (typeof availRooms === 'string') {
+    availableRooms.innerHTML = `<p>${availRooms}</p>`
+  } else {
+    availRooms.forEach(room => {
+      const image = setRoomImage(room);
+      const bidet = checkForBidet(room);
+      const cost = room.costPerNight.toFixed(2)
```

Comment:
Same comment as above: 

`You may be able to bundle all of this logic up into a function that gets all the room info and returns these in an object for you to use in your HTML below, just for SRP`

Reply:
Beautiful solution. Done.